device/globals/item.gd

```diff
@@ -479,6 +452,12 @@ func _find_sprites(p = null):
 	for i in range(0, p.get_child_count()):
 		_find_sprites(p.get_child(i))
 
+func _init():
+	add_user_signal("left_click_on_item")
```

Comment:
Same, signals can be defined with `signal` keyword.

Reply:
Thanks, didn't know that :) Updated.